include/cpuinfo.h

```diff
@@ -46,6 +46,16 @@
 	#endif
 #endif
 
+#if defined(__riscv)
+	#if (__riscv_xlen == 32)
+		#define CPUINFO_ARCH_RISCV32 1
+	#elif (__riscv_xlen == 64)
+		#define CPUINFO_ARCH_RISCV64 1
+	#else
+		#error "Unsupported RISC-V bit length."
```

Comment:
Shouldn't be an `#error`. This is a public header, and it can be included on unsupported systems.

Reply:
Fixed, this is now removed.